tutorial/index.html

```diff
@@ -0,0 +1,412 @@
+<html>
+<head>
+    <title>Akka HTTP to Persistent Actor to Akka Stream to Websocket</title>
+</head>
+<body>
+<div>
+    <h2>Getting started with this template</h2>
+
+    <p>
+        In this template you will learn some basics of
+        <strong>
+            <ul>
+                <li>Akka Http</li>
+                <li>Akka Persistence</li>
+                <li>Reactive Streams</li>
+                <li>Websockets</li>
+            </ul>
+        </strong>
+    </p>
+    <p>
+        This template consists of two parts, a theoretical and a hands-on part.
+        In the theoretical part you will learn about the techniques mentioned above by means of a fully working example.
+        In the hands-on part you will test-drive your own flow.
+        So don't mind the failing <a href="#test">tests</a> for now, you will fix them soon enough ;)
+    </p>
+    <p>
+        Next: <a href="#tutorial/1" class="shortcut">An Overview</a>
```

Comment:
Not sure if this is something we can influence, or if this is an Activator thing, but hyperlinks have an arrow picture pointing to the left, which is really weird when you want to click a link labeled "Next". It feels like we're going back, not forward.